addons/website_version/models/experiment.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+import logging
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.exceptions import Warning
+from openerp import models, fields, exceptions, SUPERUSER_ID, api
+from openerp.tools.translate import _
+import simplejson
+
+_logger = logging.getLogger(__name__)
+
+
+class Experiment_version(models.Model):
+
+    _name = "website_version.experiment_version"
```

Comment:
Note that you don't define any rec_name and you don't have any name field. I know this is quite a technical model, but probably _rec_name should be version_id.